docs/index.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: TaskHub
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
```

Comment:
It would be good if you updated the CI and codecov links. The updated ones are given in `README.md` in the root directory.